tests/integration/testdata/invoke/sam-template.yaml

```diff
@@ -10,3 +10,4 @@ Resources:
       Runtime: python3.6
       CodeUri: .
       Timeout: 600
+# 😃 UTF-8 Decoding Test
```

Comment:
Question: Without this change, does this actually fail?

Reply:
It fails if the environment is Windows and uses cp932 or some other encodings. For now, I don't think any of the test environments will fail this, which means we might need to add another one.

Reply:
Added system encoding change script for Windows in AppVeyor.